buildSrc/src/main/java/Config.kt

```diff
@@ -58,7 +60,8 @@ object Config {
     }
 
     object Sentry {
-        val SENTRY_CLIENT_NAME = "sentry.java.android"
+        val SENTRY_JAVA_CLIENT_NAME = "sentry.java"
```

Comment:
Can we name it like this to avoid confusion?
"Client" is ambiguous, "sdk name" we all understand as the following:
```suggestion
        val SENTRY_JAVA_SDK_NAME = "sentry.java"
```

Reply:
Done